Changelog.md

```diff
@@ -69,7 +69,7 @@
 
 ### Gazebo Physics 4.3.0 (2021-11-11)
 
-1. Remove unused ign_auto_headers.hh.in
+1. Remove unused gz_auto_headers.hh.in
```

Comment:
no really sure if you should modify the Changelog

Reply:
This came up before, I think we were ok with modifying it to whittle down the number of cases, so I'll just accept this one too :o